hphp/runtime/ext/mysql/ext_mysqli.php

```diff
@@ -1292,7 +1292,7 @@ public function data_seek(int $offset): void;
    * @return bool -
    */
   <<__Native>>
```

Comment:
When changing types for <<__Native>> methods, you have to change the corresponding type in the C++ implementation as well.

That said, it's probably mixed for a reason.  Does it //always// return true/false? Or might it return null or other types?